README.md

```diff
@@ -1,4 +1,8 @@
-# ecommerce-app-legends-bn-   ![check-code-coverage](https://img.shields.io/badge/code--coverage-86.38%25-green) [![Node.js CI](https://github.com/atlp-rwanda/ecommerce-app-legends-bn/actions/workflows/node.js.yml/badge.svg)](https://github.com/atlp-rwanda/ecommerce-app-legends-bn/actions/workflows/node.js.yml) [![Maintainability](https://api.codeclimate.com/v1/badges/92c706fad38b90146d03/maintainability)](https://codeclimate.com/github/atlp-rwanda/ecommerce-app-legends-bn/maintainability) [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
+<<<<<<< HEAD
```

Comment:
Resolve conflicts in Readme.md

Reply:
@ngoma3  there is infinite loading once the email is sent to the user so the user is has not known what happened after sending the request